gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
+#Tue Jan 09 20:41:01 KST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.4.1-bin.zip
-distributionSha256Sum=e7cf7d1853dfc30c1c44f571d3919eeeedef002823b66b6a988d27e919686389
```

Comment:
Did you mean to lose the hash? (Too bad it's not easier to maintain...)

Reply:
No.. It seems like the Gradle folks think it's enough to use HTTPS for downloads? If the download server is compromised than it may be likely that the `.sha256sum` file also would be.

Reply:
Added the checksum back